validmind/vm_models/result/result.py

```diff
@@ -179,13 +185,14 @@ def to_widget(self):
             self._was_description_generated = True
 
         if self.metric is not None and not self.tables and not self.figures:
-            return HTML(f"<h3>{self.result_id}: <code>{self.metric}</code></h3>")
+            return HTML(
+                f"<h3>{self.title or test_id_to_name(self.result_id)}: <code>{self.metric}</code></h3>"
```

Comment:
Given the above new property this should be:


```suggestion
                f"<h3>{self.test_name}: <code>{self.metric}</code></h3>"
```